backend/app/views/spree/admin/promotions/index.html.erb

```diff
@@ -16,33 +16,40 @@
   <div data-hook="admin_promotions_index_search">
     <%= search_form_for [:admin, @search] do |f| %>
       <div class="row">
-        <div class="col-3">
+        <div class="col-4">
           <div class="field">
             <%= label_tag :q_name_cont, Spree::Promotion.human_attribute_name(:name) %>
             <%= f.text_field :name_cont, tabindex: 1 %>
           </div>
         </div>
 
-        <div class="col-3">
+        <div class="col-2">
           <div class="field">
             <%= label_tag :q_codes_value_cont, Spree::Promotion.human_attribute_name(:code) %>
             <%= f.text_field :codes_value_cont, tabindex: 1 %>
           </div>
         </div>
 
-        <div class="col-3">
+        <div class="col-2">
           <div class="field">
             <%= label_tag :q_path_cont, Spree::Promotion.human_attribute_name(:path) %>
             <%= f.text_field :path_cont, tabindex: 1 %>
           </div>
         </div>
 
-        <div class="col-3">
+        <div class="col-2">
           <div class="field">
             <%= label_tag :q_promotion_category_id_eq, Spree::PromotionCategory.model_name.human %><br>
             <%= f.collection_select(:promotion_category_id_eq, @promotion_categories, :id, :name, { include_blank: t('spree.match_choices.all') }, { class: 'custom-select fullwidth' }) %>
           </div>
         </div>
+
+        <div class="col-2">
+          <div class="field">
+            <%= label_tag :active, 'Show Only Active' %><br>
```

Comment:
Could you add a translation for this?

Reply:
Sure, though a opposed to creating a translation I leveraged spree.active. Here is an image:
![image](https://user-images.githubusercontent.com/1048873/80431305-16bf7a80-88bf-11ea-8d4d-bd3efe378cf7.png)
I can change to a new translation if preferred, but looking at it I think inference is pretty strong.